samples/system_time/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: system time sample app
+  description: gettimeofday() and settimeofday works on zephyr project
```

Comment:
should be consistent with the `()` (either use on both function names, or neither)